### PR TITLE
Update package.yml

### PR DESCRIPTION
**Pull request by smeech**
Various additions from https://en-academic.com/dic.nsf/enwiki/3894487#sel=9:12,9:16.

### CODE REVIEW

**Comment by federico-terzi** [comment]
Hey @smeech,

Thank you for your contribution! In order to update a package, we want to create a new version of it (so that packages are version controlled). Therefore, could you please move your changes to a new 0.2.0 folder inside the `accented-words` directory? (we'll also need to bump the version inside the 0.2.0/manifest file)
If you have doubts about the structure, the idea is to basically duplicate the 0.1.0 folder into a new 0.2.0 folder and apply the changes there :)
Example: https://github.com/espanso/hub/tree/main/packages/html-utils-package

Thanks!

**Comment by smeech** [comment]
I'm sorry - you'll have to talk me through this a bit. I don't have write access to espanso/hub, and `accented-words` is currently at 1.0.0.

**Comment by federico-terzi** [comment]
@smeech Let's create a 1.1.0 version then :)

File: .gitignore
Comment (smeech) [inline .gitignore:1]: Not sure where this came from.

**Pull request closed without merging**